src/components/Overlay/__snapshots__/Overlay.spec.jsx.snap

```diff
@@ -4,29 +4,31 @@ exports[`Overlay [common] example testing should match snapshot(s) for basic 1`]
 <Portal
   portalId="1-Overlay-Portal-6"
 >
-  <CSSTransitionGroup
-    transitionAppear={false}
-    transitionEnter={true}
-    transitionEnterTimeout={300}
-    transitionLeave={true}
-    transitionLeaveTimeout={300}
-    transitionName="lucid-Overlay"
-  />
+  <TransitionGroup
+    childFactory={[Function]}
+    component="div"
+  >
+    <CSSTransition
+      classNames="lucid-Overlay"
+      timeout={300}
```

Comment:
If timeout is the same for enter and leave, you can now just pass a number for both as shown